tests/integration/pilot/common/routing.go

```diff
@@ -944,6 +945,7 @@ spec:
   trafficPolicy:
     tls:
       mode: SIMPLE
+      insecureSkipVerify: true
```

Comment:
Would this break users that donot verify server at client side

Reply:
This is a backward incompatible change, but has been sitting here ever since 1.9 release I guess. Ambient already has this enabled by default. @howardjohn was indicating that it would be a good candidate to enable by default for our compatibility version, where people get a chance to slowly migrate to this. I am trying to see how many tests would need a fix when enabling the two flags.

Reply:
Yes we should do this with https://github.com/istio/istio/pull/48449

Reply:
Agreed, I'd be hesitate to enable this by default as well given we have to modify a few of our own tests.

Reply:
This is what https://github.com/istio/istio/pull/48449 is for :-)